<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>backupDialog</class>
 <widget class="QDialog" name="backupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>265</width>
    <height>265</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>265</width>
    <height>265</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>265</width>
    <height>265</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Kodi Backup</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>164</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QListWidget" name="listDirectories_backup">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>150</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>100</height>
    </size>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Click Kodi&amp;quot;s root partition. Default is /sdcard/&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QListWidget {
    border: 2px solid #8f8f91;
    border-radius: 6px;
    border-color: black;
  
}</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listDirectories_backup2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>440</y>
     <width>150</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>100</height>
    </size>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Click Kodi&amp;quot;s thumbnail partition. Default is /sdcard/&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QListWidget {
    border: 2px solid #8f8f91;
    border-radius: 6px;
    border-color: black;
  
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="backup_label2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>410</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Thumbnails:  /sdcard/</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose partition</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>backupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>backupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listDirectories_backup</sender>
   <signal>itemDoubleClicked(QListWidgetItem*)</signal>
   <receiver>backupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>132</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
